interfaces/builtin/broadcom_asic_control.go

```diff
@@ -38,12 +38,18 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 /dev/linux-user-bde rw,
 /dev/linux-kernel-bde rw,
 /dev/linux-bcm-knet rw,
+
+/sys/devices/pci[0-9]*/**/config r,
+/sys/devices/pci[0-9]*/**/{,subsystem_}device r,
+/sys/devices/pci[0-9]*/**/{,subsystem_}vendor r,
+
+/sys/bus/pci/devices/ r,
+/run/udev/data/+pci:[0-9]* r,
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="0000:01:00.0", SUBSYSTEM=="pci", DRIVER=="linux-kernel-bde", TAG+="###CONNECTED_SECURITY_TAGS###"
```

Comment:
The above rules address my concerns as well.
To some extent, IMHO, The following rule is probably good enough in this case
`DRIVER=="linux-kernel-bde", TAG+="###CONNECTED_SECURITY_TAGS###"`
Let's finalise with yours proposal
`SUBSYSTEM=="pci", DRIVER=="linux-kernel-bde", TAG+="###CONNECTED_SECURITY_TAGS###"`